### ISSUE TITLE
Export Questions List

### ISSUE THREAD

**Issue by qodzero** [sdaps/sdaps#190]
Hello again, I would like to know if its possible to export a list of all the questions in my questionnaire as well as the available options. I've been going through the gui module and the report module as I think its somewhere in there otherwise how does the gui for example know the questions. Unfortunately I dont know the first thing about GTK so things got confusing really fast.

**Comment by benzea** [comment]
Should be pretty simple to do. It may require understanding the "buddies" concept though.

In principle, you can just look at `survey.questionnaire`. That Object has a list of `qobjects` (`survey.questionnaire.qobjects`) and if you iterate over these you will get each question/header. Depending on the objects type you can then dump the relevant information (e.g. "title" or "question") and also query the boxes.

**Comment by qodzero** [comment]
I ended up using the data in tne `survey.sqlite` db which has something that looks like what you mentioned in your comments..thanks

**Comment by benzea** [comment]
Why close the bug? The feature seems still reasonable to add to SDAPS upstream.

**Comment by benzea** [comment]
But yeah, survey.sqlite contains all the information encoded in json.

**Comment by qodzero** [comment]
Well, I'll share my code and make a pull request..I was thinking maybe make it an option in the csv export function like how sdaps exports texfield images optionally?

**Comment by benzea** [comment]
Yup, extending CSV export would work. Though I guess it might make more sense to have an `export_questions` or so command rather than a flag for the CSV exporter.

i.e. we probably want to only export the questions or the data, not both.

**Comment by qodzero** [comment]
Ok, that makes sense
